pycheribuild/projects/run_fvp.py

```diff
@@ -472,6 +467,9 @@ def setup_config_options(cls, **kwargs):
                                                            help="When set rsync will be used to update the image from "
                                                                 "the remote server prior to running it.")
         cls.ssh_port = cls.add_config_option("ssh-port", default=cls.default_ssh_port(), kind=int)
+        cls.more_tcp = cls.add_config_option("more-tcp", kind=str, default="",
+                                             help="Additional TCP bridge ports beyond ssh/22; comma separated list of "
+                                                  "items of the form [hostip:]port=[guestip:]port")
```

Comment:
Does the FVP accept prefixing with an IP address?

Reply:
As per https://developer.arm.com/documentation/100964/1113/Introduction/Network-set-up/User-mode-networking .  If there's a better source of truth, I'm all ears.

Reply:
I am pleasantly surprised :)